app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+
+  def is_owned_by?(user)
```

Comment:
I'd put this in a module in models, or in a "concern" - helpers is meant for view helpers.